Source/Examples/WPF/WpfExamples/Examples/AnimationsDemo/AnimationExtensions.cs

```diff
@@ -0,0 +1,112 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="MainViewModel.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace AnimationsDemo
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using System.Windows.Media.Animation;
```

Comment:
The only reason I have animation is because of the EasingFunctionBase because I needed an easy way to show different values. CalculateEaseValues can be edited so it has it's own implementation and can be portable :)